app/models/katello/glue/candlepin/owner.rb

```diff
@@ -36,7 +36,9 @@ def content_access_mode
       end
 
       def simple_content_access?
```

Comment:
this probably isn't working since you need to add the `cached` keyword argument to this method (default to true)